pkg/controller/types/networkpolicy.go

```diff
@@ -37,6 +37,18 @@ func (meta *SpanMeta) Has(nodeName string) bool {
 	return meta.NodeNames.Has(nodeName)
 }
 
+// TierPriority specifies the relative ordering among Tiers. A lower
+// TierPriority indicates higher precedence.
+type TierPriority uint32
+
+const (
+	TierEmergency TierPriority = iota + 1
```

Comment:
How these tier names are decided? I am not sure SecurityOps, InterTenant, etc. fit into K8s.

Reply:
@McCodeman : thoughts?

Reply:
actually these names were suggested by @mccodeman :)

Reply:
I still feel tier name like InterTenant is hard to understand in the K8s context.

Reply:
I think the inter tenant was used in Cody’s prior job to describe namespace owners when the namespace acted as tenancy separation. but I Platform was also suggested by Cody in place of inter tenant. I will make the change to platform. And maybe Cody can provide additional comments